recipes/dbus/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+from conans import CMake, ConanFile, tools
+import os
+
+
+class DbusConan(ConanFile):
+    name = "dbus"
+    license = "AFL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/dbus"
+    description = "D-Bus is a simple system for interprocess communication and coordination."
+    topics = ("conan", "dbus")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+            "with_x11": [True, False], 
+            "with_glib": [True, False],
+            "disable_assert": [True, False],
+            "disable_checks": [True, False],
+            "enable_ansi": [True, False],
+            "enable_containers": [True, False],
+            "enable_stats": [True, False],
+            "enable_verbose_mode": [True, False],
```

Comment:
Does this increase dbus' verbosity? Or only the verbosity of the build process?

Reply:
I just added all of this lines from CMakeCache and ccmake, so I don't really know what they mean.

Reply:
Please make sure to only add options which are relevant for dbus consumers